src/ray/protobuf/job_info_access.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+import "src/ray/protobuf/gcs.proto";
+
+message GcsJobInfo {
+}
```

Comment:
```
message AddJobRequest {
  JobTableData data;
}
```
The naming convention is to name the request and reply of `Foo` method as `FooRequest` and `FooReply`.